### PR TITLE
Added flash message

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

File: Resources/translations/FSiAdminSecurity.en.yml [hunk 1]
```diff
@@ -63,6 +63,8 @@ admin:
     batch_action:
       delete: Delete
       password_reset: Reset password
+    message:
+      delete_current_user: Cannot delete your own user
```
Comment (szymach) [inline Resources/translations/FSiAdminSecurity.en.yml]: *your

Comment (chives) [inline Resources/translations/FSiAdminSecurity.en.yml]: "your own something" is perfectly correct and widely used phrase

**Comment by chives** [comment]
@fsi-open

File: Resources/translations/FSiAdminSecurity.en.yml (hunk 1 above)
Comment (JarekW) [inline Resources/translations/FSiAdminSecurity.en.yml]: "yourself", but whatever for me. :+1:

**Pull request merged**